<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(168, 168, 168);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="loginPageBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 58, 110);
border-width: 0px;</string>
        </property>
        <property name="text">
         <string>登录页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="patientInfoPageBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(128, 65, 255);
color: rgb(255, 255, 255);
border-width: 0px;</string>
        </property>
        <property name="text">
         <string>患者信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deviceBindBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(47, 255, 196);
border-width: 0px;</string>
        </property>
        <property name="text">
         <string>设备绑定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="wavePageBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 58, 110);
border-width: 0px;</string>
        </property>
        <property name="text">
         <string>波形数据</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="otherDataBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 58, 110);
border-width: 0px;</string>
        </property>
        <property name="text">
         <string>其他数据</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(255, 58, 110);</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="loginPage">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>账号</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="accountEdit"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>密码</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="passwordEdit">
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>登录</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="statusLabel">
                <property name="text">
                 <string>状态：未登录</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="patientPage">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(128, 65, 255);</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,1">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>姓名</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="nameEdit"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>性别</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="sexEdit"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>年龄</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ageEdit"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>身份证号</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="idNumEdit"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="insertBtn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>插入</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QTableView" name="patientTable"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QPushButton" name="updateBtn">
                <property name="text">
                 <string>刷新</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="modifyBtn">
                <property name="text">
                 <string>修改</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="deleteBtn">
                <property name="text">
                 <string>删除</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="devicePage">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(47, 255, 196);</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>患者</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableView" name="patiBindTable"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>设备</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableView" name="deviceBindTable"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QPushButton" name="updateBindBtn">
              <property name="text">
               <string>更新数据</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bindBtn">
              <property name="text">
               <string>绑定</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="unBindBtn">
              <property name="text">
               <string>解绑</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>绑定</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="patideviBindTable"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="wavePage">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(67, 67, 67);</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11" stretch="0,1,1,1">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="1,1,0,0,0">
            <item>
             <widget class="QDateTimeEdit" name="dateTimeEdit"/>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="dateTimeEdit_2"/>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton">
              <property name="text">
               <string>历史</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_2">
              <property name="text">
               <string>实时</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_8">
              <property name="text">
               <string>更新</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true"/>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true"/>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="otherPage">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 182, 92);</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QRadioButton" name="radioButton_3">
              <property name="text">
               <string>HR</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_4">
              <property name="text">
               <string>ST</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_5">
              <property name="text">
               <string>NIBP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_6">
              <property name="text">
               <string>SPO2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_7">
              <property name="text">
               <string>TEMP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_8">
              <property name="text">
               <string>RESP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_9">
              <property name="text">
               <string>CO2</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListView" name="listView"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
